formfactory/__init__.py

```diff
@@ -1,28 +1,65 @@
 from django.conf import settings
 
 
+class AppFields(object):
+    DJANGO = "django.forms.fields"
+    FORMFACTORY = "formfactory.fields"
+
+class AppWidgets(object):
+    DJANGO = "django.forms.widgets"
+    FORMFACTORY = "formfactory.widgets"
+
 SETTINGS = getattr(settings, "FORMFACTORY", {
     "field-types": [
-        "BooleanField", "CharField", "ChoiceField", "DateField",
-        "DateTimeField", "DecimalField", "EmailField", "FileField",
-        "FloatField", "GenericIPAddressField", "IntegerField",
-        "MultipleChoiceField", "SlugField", "SplitDateTimeField", "TimeField",
-        "URLField", "UUIDField"
+            ("%s.BooleanField" % AppFields.DJANGO, "BooleanField"),
```

Comment:
You're accessing a class attr on an uninitialized class. It will work but looks odd. What is the point of these classes?

Reply:
Honestly, was just a shorter bit to copy paste. Will make all the keys full strings.